lib/axon_onnx/deserialize.ex

```diff
@@ -1383,6 +1383,12 @@ defmodule AxonOnnx.Deserialize do
           layer = Axon.layer([inp, min, max], fun, %{}, output_name, layer_op: :clip)
           updated_axon = Map.put(axon, output_name, layer)
           {updated_axon, used_params}
+
+        {%Axon{} = inp, %Nx.Tensor{} = min, %Nx.Tensor{} = max} ->
```

Comment:
Can you add a clause to check to see if `inp` is constant, if so we can constant fold:

```suggestion
        {%Axon{op: :constant, opts: [value: v]} = inp, %Nx.Tensor{} = min, %Nx.Tensor{} = max} ->
          new_value = Nx.clip(v, min, max)
          layer = Axon.constant(new_value, name: output_name)
          updated_axon = Map.put(axon, output_name, layer)
          {updated_axon, used_params}
```

Reply:
yeah, i can certainly do that -- i have a nagging thought, though: should a deserializer concern itself with constant folding _at all_? three things jump out at me:
1. it's a model optimization. though it's not too _hard_ to do here, since we're already crawling the structure, is it _right_ to be doing it here? why not give axon a model optimizer (that maybe just does constant folding at first?)
2. it might violate my expectations if i'm loading the model for some reason that i _don't want_ the constants folded for -- maybe just to save it in another format?
3. a _really_ easy way to test this big, intricate thing (and the serializer along with it) would be to eat our own dogfood here and have tests that deserialize reasonably complex mainstream models, serialize them again and compare. in-line optimizations of the model during deserialization could make this harder.

Reply:
oh, okay -- that makes a lot of sense. there's a lot of moving parts, and it's all very new, so some things are gonna have to wait for advancements in other places. i'll add as you suggest. btw, is there a slack or something where a person could ask stupid questions (besides here?) that'd make it easier to get up to speed. ;)

Reply:
If you sign up for the Erlef: https://members.erlef.org/join-us

Then you can join the EEF slack and join the ML WG slack channel where all of the maintainers work